index.js

```diff
@@ -5,13 +5,14 @@ const messages = document.getElementsByClassName('message');
 const tooHighMessage = document.getElementById('too-high');
 const tooLowMessage = document.getElementById('too-low');
 const maxGuessesMessage = document.getElementById('max-guesses');
-const numberOfGuessesMessage = document.getElementById('num-of-guesses');
+const numberOfGuessesMessage = document.getElementsByClassName('num-of-guesses');
+console.log(numberOfGuessesMessage);
```

Comment:
You might have noticed that when you log this you get back an `HTMLCollection []`. This is a type of list or array. `getElementsByClassName` looks at the HTML file and tries to grab every element with the class name you give it (in this case `num-of-guesses`. All the matches it finds it's going to put in that `HTMLCollection` list. But if it doesn't find any, then the list will just be empty. You can try opening the list in the Dev Console and you'll see that right now what's being logged is empty. Try to look into why `document.getElementsByClassName('num-of-guesses')` returned an empty list. Also do we want a list of elements here or just a single one?